index.js

```diff
@@ -61,7 +67,10 @@ const envy = (input) => {
         return filterObj(camelizedGlobalEnv, camelizedExampleEnvKeys);
     }
 
-    if (checkMode(envPath, permissionMask) !== ownerReadWrite) {
+    if (isWindows() && checkMode(envPath, permissionMask) !== windowsPermission) {
+        throw new Error(`File permissions are unsafe. Make them 666 '${envPath}'`);
```

Comment:
I'm not aware of a command line way. I've been doing it from windows configuration panels but indeed it would be nice to do it with the command line. I wonder if you can call that command from bash or if you want to open `Powershell` or something similar.

Reply:
Do you want me to update the PR with the "group" and "others" write bits off?